Integrations/IronDefense/IronDefense.yml

```diff
@@ -0,0 +1,155 @@
+category: Network Security
+commonfields:
+  id: IronDefense
+  version: -1
+configuration:
+- display: IronAPI Host/IP
+  name: ironAPIHost
+  required: true
+  type: 0
+- defaultvalue: '6942'
+  display: IronAPI Port
+  name: ironAPIPort
+  required: true
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- defaultvalue: '60'
+  display: Request Timeout (Sec)
+  name: requestTimeout
+  required: false
+  type: 0
+description: Integrates with IronDefense
```

Comment:
add a more informative product description (what are the capabilities of the integration)